src/renderer/prefComponents/common/fontTextBox/index.vue

```diff
@@ -83,12 +84,14 @@ export default {
         : fontFamilies
       callback(results)
     },
+
     handleSelect (value) {
-      if (/^[_A-z0-9]+((-|\s)*[_A-z0-9])*$/.test(value)) {
+      if (/^[^\s]+((-|\s)*[^\s])*$/.test(value)) {
```

Comment:
@Jocs I think you forgot to edit the `preferences/schema.json` file to allow all font families. When doing so, you should remove the regex completely and implement a proper XSS protection on the editor (renderer) to prevent bad font names. The attack vector is no very high but it's still a file-based Cross-Site Scripting when we don't verify the font family.